src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/readthreading/ReadThreadingAssembler.java

```diff
@@ -396,7 +396,7 @@ List<Haplotype> findBestPaths(final Collection<T> graphs, final Map<T, AssemblyR
                         resultSet.add(h);
                     }
 
-                    logger.log(debug ? Level.INFO : Level.OFF, () -> "Adding haplotype " + h.getCigar() + " from graph with kmer " + assemblyResult.getKmerSize());
+                    logger.log(debug ? Level.INFO : Level.TRACE, () -> "Adding haplotype " + h.getCigar() + " from graph with kmer " + assemblyResult.getKmerSize());
```

Comment:
So the issue is that calling logger.log(Level.OFF, message) will print that message in every case if you turn the logger off? If thats true I actually want you to revert these to the way they were before they were recently refactored (which i reviewed and didn’t catch here)… replace them with the previous statements 
```
if(debug) {
            logger.info( "Not using kmer size of " + kmerSize + " in read threading assembler because reference contains non-unique kmers");
}
```